### ISSUE TITLE
Python 2.7 missing requirements. 

### ISSUE THREAD

**Issue by xando** [vmprof/vmprof-server#6]
Despite 3.4 being in travis (no sure how this got there) Python 2.7 is still default deploy target. 

Missing requirements after merging 67bf0d88817046bcc17a71e41687a3d7e0788461

```
(.virtualenv) vagrant@vmprof:/vagrant/vmprof-server$ python manage.py runserver 0.0.0.0:8000
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/vagrant/vmprof-server/.virtualenv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/vagrant/vmprof-server/.virtualenv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 312, in execute
    django.setup()
  File "/vagrant/vmprof-server/.virtualenv/local/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/vagrant/vmprof-server/.virtualenv/local/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/vagrant/vmprof-server/.virtualenv/local/lib/python2.7/site-packages/django/apps/config.py", line 198, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/vagrant/vmprof-server/vmlog/models.py", line 4, in <module>
    import lzma
ImportError: No module named lzma
```

**Comment by planrich** [comment]
Summary from the IRC:

yes, I switched vmprof-server to python 3.4. Is that an issue? I think sooner or later that should be done anyway (which is a good incentive to push pypy 3.x forward :)

Additionally vmprof-server has a new structure (I moved the generic 'server' to 'vmprofile'. There might be some issues while migrating (need to rename the database tables).

Another thing that I changed was debug = True to False in production, and we should not let django deliver the javascript and css files, but configure nginx to deliver them.

Of course I can revert some of the changes if there is good reason to do so.
